openbb_platform/core/openbb_core/app/service/hub_service.py

```diff
@@ -1,27 +1,38 @@
 """Hub manager class."""
 
 from typing import Optional
+from warnings import warn
 
 from fastapi import HTTPException
 from jose import JWTError
 from jose.exceptions import ExpiredSignatureError
 from jose.jwt import decode, get_unverified_header
 from openbb_core.app.model.abstract.error import OpenBBError
 from openbb_core.app.model.credentials import Credentials
-from openbb_core.app.model.hub.features_keys import FeaturesKeys
 from openbb_core.app.model.hub.hub_session import HubSession
 from openbb_core.app.model.hub.hub_user_settings import HubUserSettings
 from openbb_core.app.model.profile import Profile
 from openbb_core.app.model.user_settings import UserSettings
 from openbb_core.env import Env
-from pydantic import SecretStr
 from requests import get, post, put
 
 
 class HubService:
```

Comment:
Should `connect()` method become private and called on the constructor?

Reply:
I did it this way to become obvious that you can `connect` or `disconnect`